SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DormancyTests/InitiallyDormantMapActorDeletionTest.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "InitiallyDormantMapActorDeletionTest.h"
+
+#include "InitiallyDormantMapActorTest.h"
+
+#include "DormancyTestActor.h"
+
+// Tests that an initially dormant startup actor, which remains dormant, can be deleted on the client.
+
+AInitiallyDormantMapActorDeletionTest::AInitiallyDormantMapActorDeletionTest()
```

Comment:
Nice, I assume this didn't pass without your change, and now passes with the change?

Reply:
Yeah I tested it without the fix and with the fix, fails without fix.